Base/usr/share/man/man1/modload.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+modload - loads a kernel module
+
+## Synopsis
+
+```**sh
+$ modload <path>
+```
+
+## Description
+
+Loads a kernel module specified by *path*.
```

Comment:
And then in description, you either do the same ("Load a kernel module specified by *path*") or go "`modload` loads a kernel module..."